toolkit/tools/internal/versioncompare/versioncompare.go

```diff
@@ -149,6 +156,11 @@ func (v *TolerantVersion) parse(versionString string) {
 	// Run again if we have a release version as well
 	if releaseSubstring != "" {
 		rawComponents = componentRegex.FindAllString(releaseSubstring, -1)
+		// If no epoch is set in the version, apply an epoch of 0 so all versions have one.
```

Comment:
Can a release have a separate epoch? I think this is just for parsing the ***-<subrelease> part of the version.

Reply:
Ah, good catch. I'll remove this from the release parsing.